src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPageObject.java

```diff
@@ -243,6 +267,69 @@ public void banUser(String userName) {
     PageObjectLogging.log("clickBanUser", userName + " ban modal is closed", true);
   }
 
+  public void clickOpenUserMessageWall(String userName) {
```

Comment:
True removed user name from click and added verifyUserIsOnCorrectUserProfilePage method to tests :)